host/tests/dsClasses/dsFPD_test_suite.yml

```diff
@@ -13,12 +16,17 @@ dsFPD:  # Prefix must always exist
                 name: "L2 dsFPD - Sink"
```

Comment:
In the dsFPD_test_suite.yml please add the menu names for L1 and L2 also

Reply:
updated

Reply:
updated